mllib/src/main/scala/org/apache/spark/mllib/rdd/SlidingRDD.scala

```diff
@@ -42,8 +42,8 @@ class SlidingRDDPartition[T](val idx: Int, val prev: Partition, val tail: Seq[T]
  * @param windowSize the window size, must be greater than 1
  * @param step step size for windows
  *
- * @see [[org.apache.spark.mllib.rdd.RDDFunctions.sliding(Int, Int)*]]
- * @see [[scala.collection.IterableLike.sliding(Int, Int)*]]
+ * @see `org.apache.spark.mllib.rdd.RDDFunctions.sliding(Int, Int)*`
```

Comment:
Heh, OK. I don't think it has any meaning in a hyperlink or javadoc syntax, and this isn't either one anyway, but it's OK to leave it.